src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -735,7 +735,12 @@
   <Node Name="BoundMultipleLocalDeclarations" Base="BoundStatement">
     <Field Name="LocalDeclarations" Type="ImmutableArray&lt;BoundLocalDeclaration&gt;"/>
   </Node>
- 
+
+  <Node Name="BoundUsingLocalDeclarations" Base="BoundMultipleLocalDeclarations">
+    <Field Name="DisposeMethodOpt" Type="MethodSymbol" Null="Allow"/>
+    <Field Name="IDisposableConversion" Type="Conversion?" Null="Allow"/>
```

Comment:
I originally suggested using a nullable here, but it looks like most other places just use Conversion.NoConversion, so we should probably not make this nullable.